packages/jaeger-ui/src/components/TracePage/ScrollManager.tsx

```diff
@@ -184,7 +189,7 @@ export default class ScrollManager {
 
       // If there are hidden children, scroll to the last visible span
       if (childrenAreHidden) {
-        let isFallbackHidden: boolean | TNil;
+        let isFallbackHidden: boolean;
```

Comment:
I introduced it as `?boolean` which was necessary according to flow. Else it says:
`  ^ possibly uninitialized variable [1] is incompatible with boolean [2].`
I imagine we'll continue to find some `TNil` over-usage since I did `s/?\([a-zA-Z]\{1,\}\)/\1 | TNil/` pretty liberally.